### ISSUE TITLE
text prompt for "position(s) modified

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1085]
Greyed out text
e.g. K23
(and whatever we would type for more than 1)
e.g. K23, S54
(is that correct? comma separated?)

**Comment by Antonialock** [comment]
hmm, I guess if you are confident two modifications exist at the same time (and are of the same type) then you could comma separate? i would probably pipe-separate though

**Comment by mah11** [comment]
I think comma-separated would be correct, because that would mean both/all are modified at once. Pipe-separated entries would not imply that they exist at the same time, and would be the same as making two separate extensions. So you would only *need* to enter two or more for the same extension if they're all-at-once and therefore comma-separated.

**Comment by ValWood** [comment]
Maybe the text needs to say

e.g. K23,S54 (comma indicates simultaneous modification)

Or we just change to allow only independent modifications until we have a case where we need to show simultaneous
?

**Comment by Antonialock** [comment]
Well, we also use extensions to show simultaneous...for example say you have a phosphorylated threonine and a phosphorylated serine during DNA damage response, you couldn't comma separate because they would need to be entered separately

**Comment by mah11** [comment]
Why wouldn't "added_during cellular response to DNA damage stimulus , residue S23,T25" work?

**Comment by Antonialock** [comment]
I always go for the specific modification terms i.e. o-phospho-L-serine / O-phospho-L-threonine.

I guess you could go for 'phosphorylated residue' as long as you don't have two very different modifications i.e. a phosphorylated residue and a farnesylated residue (or whatever)

**Comment by mah11** [comment]
OK, that makes sense. I guess there are two options: lose the info that they're simultaneous, or add a redundant annotation to a more generic term to put the compound extension on. The latter would be pretty tedious for anyone to do.

I also don't remember needing to annotate any simultaneous residue modifications anyway, so maybe it's simplest just to collect one at a time.

**Comment by ValWood** [comment]
Smae here. Lets go with the simplest option, until we have a use case

**Comment by kimrutherford** [comment]
> Lets go with the simplest option, until we have a use case

Is this something I need to change?  If so, could you tell me where this text is?  Sorry, I've lost track.

**Comment by ValWood** [comment]
Ah, ok I thought I commented on here yesterday. You might find a random comment in another ticket.

The decision is, for now, only allow a single residue to be specified at once. Multiple independent residues = independent annotations. If we end up having use cases where we *really* want to capture co-occuring modifications we can revisit this.

**Comment by kimrutherford** [comment]
> The decision is, for now, only allow a single residue to be specified at once.

Does that need to be done before releasing the changes?

**Comment by ValWood** [comment]
I don't think so as its working. We can override if necessary and  will  put as next (doesn't seem too tricky so can be doon soon after release)?

**Comment by ValWood** [comment]
e.g. K23

**Comment by kimrutherford** [comment]
I've changed the config, it will be live in the morning.
